realm/realm-library/src/main/java/io/realm/RealmSchema.java

```diff
@@ -156,4 +177,64 @@ private void checkHasTable(String className, String errorMsg) {
             throw new IllegalArgumentException(errorMsg);
         }
     }
+
+    ColumnInfo getColumnInfo(Class<? extends RealmObject> clazz) {
+        final ColumnInfo columnInfo = columnIndices.getColumnInfo(clazz);
```

Comment:
Then we crash, I just moved the method to this class. AFAIK the callers of this method guarantee that it is non-null